server/server/users/tests/factories.py

```diff
@@ -34,55 +34,25 @@ class Meta:
         django_get_or_create = ["username"]
 
 
-class ConsumerFactory(DjangoModelFactory):
-
-    username = Faker("user_name")
-    email = Faker("email")
-    name = Faker("name")
-
-    @post_generation
-    def password(self, create: bool, extracted: Sequence[Any], **kwargs):
-        password = (
-            extracted
-            if extracted
-            else Faker(
-                "password",
-                length=42,
-                special_chars=True,
-                digits=True,
-                upper_case=True,
-                lower_case=True,
-            ).evaluate(None, None, extra={"locale": None})
-        )
-        self.set_password(password)
-
+class ConsumerFactory(UserFactory):
```

Comment:
Mixins for all

Reply:
problematic. Userfactory attributes & decorators are used in djangoModelFactory. causes issues.